src/scss/grommet-core/_objects.icon.scss

```diff
@@ -88,7 +88,7 @@
   }
 
   @each $status, $color in $brand-status-colors {
-    &-#{$status} {
+    .control-icon-#{$status} {
```

Comment:
would you mind to add the grommet namespace here?

`.#{$grommet-namespace}control-icon-#{$status} {`